minio/api.py

```diff
@@ -44,7 +44,8 @@
 
 # Internal imports
 from . import __title__, __version__
-from .compat import urlsplit, range, urlencode, basestring
+from .compat import (urlsplit, queryencode,
```

Comment:
This is a good idea @Yurmusp will test it out and see if there are any problems for 3.x